app/Components/Header/Type/Header.type.ts

```diff
@@ -1,3 +1,3 @@
-import { HeaderPropsInterface } from '../Interfaces/header-props.interface';
+import { HeaderPropsInterface } from '../Interfaces/Header-props.interface';
```

Comment:
interfaces folder should be lower case also name of interface shold be header-props.interface.ts